src/DotNest.Core.SDK.Web/appsettings.json

```diff
@@ -5,5 +5,13 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "OrchardCore_DataProtection_Azure": {
+    "CreateContainer": true
+  },
+  "OrchardCore_Media_Azure": {
+    "ContainerName": "media",
+    "BasePath": "{{ ShellSettings.Name }}",
+    "CreateContainer": true
+  },
```

Comment:
Use this instead:
 ``` json
  "OrchardCore_DataProtection_Azure": {
    "ConnectionString": "UseDevelopmentStorage=true",
    "ContainerName": "dotnestdataprotection"
  },
  "OrchardCore_Media_Azure": {
    "ConnectionString": "UseDevelopmentStorage=true",
    "ContainerName": "dotnestmedia"
  },
```

Reply:
And also this shouldn't be here. Please follow what I wrote in [this comment.](https://github.com/Lombiq/DotNest-Core-SDK/pull/1#pullrequestreview-1027291776). (Create an appsettings.Development.json file)

Reply:
You also forget to add autosetup.

Reply:
In your last request you suggested that I should remove appsettings.development.json

Reply:
By "This can be deleted, it is the same in appsettings.json." I meant the lines.